src/app/data-explorer/selection-expanded-dialog/selection-expanded-dialog.component.html

```diff
@@ -0,0 +1,39 @@
+<!--
+Copyright 2022-2023 University of Oxford
+and Health and Social Care Information Centre, also known as NHS Digital
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+SPDX-License-Identifier: Apache-2.0
+-->
+
+<div mat-dialog-title class="dialog-title">
+  <h1>Selected data elements</h1>
+  <button mat-icon-button aria-label="close dialog" mat-dialog-close>
+    <mat-icon>close</mat-icon>
+  </button>
+</div>
+<mat-dialog-content>
+  <p>{{ (selectedElements$ | async)?.length }} elements selected</p>
+  <div *ngFor="let element of selectedElements$ | async">
+    <mat-checkbox checked="checked" (click)="deselectElement(element.id)"> </mat-checkbox>
+    <span>{{ fullPathFor(element) }}</span>
+  </div>
+</mat-dialog-content>
+<mat-dialog-actions align="end">
+  <mdm-data-element-multi-select
```

Comment:
I've noticed that the "Current selection" box has the Clear button then the "With Selected..." button, but in this dialog the order is swapped. I think it would be nicer if the button order was consistent, but I have no strong opinion on which way round both should be.